src/darwin/Framework/CHIP/MTRAttributeSpecifiedCheck.h

```diff
@@ -16,10 +16,6 @@
  *    limitations under the License.
  */
 
-#pragma once
```

Comment:
So these are not needed because the expectation is people use #import which handles this?

Reply:
Yes